sample_information_validator.py

```diff
@@ -1,20 +1,20 @@
 #!/usr/bin/env python
-"""
-Validates the user submitted sample information sheet
-so that no errors are propagated downstream.
-"""
+
+# load libraries
 from __future__ import print_function
 import argparse
 from openpyxl import load_workbook
 import coloredlogs
 import logging
+import re
+import couchdb
+import numbers
+import decimal
```

Comment:
I think the `numbers` and `decimal` modules are not used, right?

Reply:
actually,` numbers` seems to be used: 
`218 if not isinstance(self.access_sample_info_sheet[checkNumbers].value, **numbers.Number**):`